### ISSUE TITLE
Void jars not interacting with export, storage bus, or essentia provider.

### ISSUE THREAD

**Issue by Sipiri** [Nividica/ThaumicEnergistics#227]
I've been trying to find a way to pump essentia into a void jar. Export and storage bus blocks will behave as expected with regular warded jars, but refuse to cooperate with void jars. Whether the source be from a storage cell, an alembic, or a warded jar- no matter what the priority settings are nor whether I have the 'void essentia' option toggled, nothing works.

My furious google searches have yielded nothing and there seems to be no solid information on how void jars are meant to interact with this mod, so it may be due to my own incompetence.

Running version 0.8.10.9

**Comment by AltonV** [comment]
I too have trouble voiding essentia using void jars and storage buses.
Essentia keeps piling up in my alchemical furnace & alembics.

Version 0.8.10.10

**Comment by katzenpapst** [comment]
Can confirm, same here...

**Comment by PhoenixtheII** [comment]
Same here with thaumicenergistics-0.8.10.10 (FTB Infinity)

Temporary workaround: 

https://www.dropbox.com/s/bkgp4tjp83m9vbd/Screenshot%202015-08-20%2018.41.29.png?dl=0

Normal jar + autonomous activator that is sneaking + right clicking... :)

**Comment by cactusbee** [comment]
we also got this bug in our custom modpack. both the oblivion and void jars won't even accept, let alone void, the essentia from the storage bus. i have tried everything including changing the layout of the parts, creating brand new jars, setting priority values high and low, putting the bus on its own separate line of cable, and ensuring there is indeed a channel to the storage bus. 

curiously, it worked fine as i was assembling the prototype version of the machine. the final design, though, is exactly the same as when i was testing. the only difference is that i moved the machine parts to a different area. when i created the prototype, we were on an version of TE. now that we have updated to the latest one, this began happening. i am not sure if it is a coincidence or not.

http://puu.sh/jILvB/5789ee90cc.jpg here is a picture of the relevant (i think) parts of the machine. like one of the posters above me, i came up with a workaround solution in the form of a golem who likes to sneak right click :)
